src/njoy21/input/LEAPR/Card1.hpp

```diff
@@ -0,0 +1,16 @@
+class Card1 {
+public:
+  #include "njoy21/input/LEAPR/Card1/Nout.hpp"
+  Argument < Nout > nout;
```

Comment:
It's actually not exactly like the common"one, since this nout is not dependent on a nin value